lib/src/print_strategies.dart

```diff
@@ -0,0 +1,26 @@
+import 'logging_bugfender.dart';
+
+/// Defines how [LoggingBugfenderListener] prints.
+abstract class PrintStrategy {
```

Comment:
I dont get it, why is it an interface and not just an enum?

Reply:
This strategy should have an abstract method that we actually can have various strategies on - creating the String log message.

Reply:
Initially I thought of adding some config options, for example to `ColoredTextPrintStrategy`. But yeah, this should be an enum.

Reply:
No. Refer to my comment below and [what strategies are](https://en.wikipedia.org/wiki/Strategy_pattern). :)

Reply:
I added my comment while yours @Albert221 was not visible. Sorry :)